apps/ehr/test/e2e-utils/resource-handler.ts

```diff
@@ -198,6 +201,69 @@ export class ResourceHandler {
         console.error('❌ QuestionnaireResponse not created', error);
       }
     }
+
+    if (!this.insurancePlan) {
+      try {
+        this.insurancePlan = await this.apiClient.fhir.create({
```

Comment:
Looks like that's outdated code. We are currently using `createSamplePrebookAppointments` and `createSampleTelemedAppointments` from `utils` package. I think we should add insurance handling in those zambdas too.